src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,28 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
```

Comment:
Don't begin class or method implementation with empty line. Remove the empty lines between the package statement and the class declaration.